ipaserver/install/server/upgrade.py

```diff
@@ -1375,7 +1375,7 @@ def remove_ds_ra_cert(subject_base):
 def fix_trust_flags():
     root_logger.info('[Fixing trust flags in %s]' % paths.HTTPD_ALIAS_DIR)
 
-    if sysupgrade.get_upgrade_state('http', 'fix_trust_flags'):
+    if sysupgrade.get_upgrade_state('http', 'fix_trust_flags2'):
```

Comment:
Rob is right that you should probably have a new state to do this, this one was supposed to do some changes and you're adding some more on top of that.